tasks-dev/browsersync.js

```diff
@@ -0,0 +1,56 @@
+const gulp = require('gulp');
+const browserSync = require('browser-sync');
+
+const config = require('../config');
+const pathBuilder = require('../pathBuilder');
+
+
+/**
+ * `browser-sync` Task
```

Comment:
Mild OCD alert: If the task is called `browser-sync` the filename should be the same (with a dash) as that’s what all the other tasks do too 😉